bin/templates/scripts/cordova/lib/prepare.js

```diff
@@ -275,12 +275,59 @@ function handleOrientationSettings (platformConfig, infoPlist) {
     }
 }
 
+// Make sure only update properties from our target project
+function updateBuildPropertyLocal(proj, displayName, prop, value, build) {
+    try {
+        // Check if we have a valid target - during prepare we do not have it
+        var target = proj.pbxTargetByName(displayName);
```

Comment:
Does this definitely use the display name, and not the full name? In Cordova apps those are usually the same, but it's not guaranteed (and we provide a way to change the display name).

Reply:
Can you give me an example how to change the name - I will test that case.

Reply:
In config.xml:

```xml
<name short="MyApp">My Full Application Name</name>
```

In that example, `"MyApp"` would be the short name (which is what you're putting into the `displayName` variable).

Reply:
Yes you were right - I have fixed the pbx target retrieval (tried both variants)